core/opendaq/signal/include/opendaq/input_port_impl.h

```diff
@@ -188,7 +188,7 @@ ErrCode GenericInputPortImpl<Interfaces...>::connect(ISignal* signal)
 
         auto signalPtr = SignalPtr::Borrow(signal);
         Bool accepted;
-        err = this->acceptsSignal(signalPtr, &accepted);
+        err = this->GenericInputPortImpl::acceptsSignal(signalPtr, &accepted); // TODO ???
```

Comment:
I'm not 100% sure that we should be even calling `acceptsSignal` here. We should probably let users connect any signal, as it can later be reconfigured. @dejancrn thoughts?

Reply:
@aljazfrancic We should remove the `acceptsSignal` call here and allow users to connect signals that are not accepted as reasoned above.